packages/transactions/src/__tests__/durable-nonce-test.ts

```diff
@@ -129,6 +129,29 @@ describe('assertIsDurableNonceTransaction()', () => {
             assertIsDurableNonceTransaction({ ...durableNonceTx });
         }).not.toThrow();
     });
+    it('does not throw when the nonce authority is a writeable signer', () => {
```

Comment:
```suggestion
    it('does not throw when the nonce authority is a writable signer', () => {
```

Reply:
I don't even have the excuse of saying that's an English spelling!